Source/WebInspectorUI/UserInterface/Controllers/NetworkManager.js

```diff
@@ -1564,12 +1564,33 @@ WI.NetworkManager = class NetworkManager extends WI.Object
         target.NetworkAgent.loadResource(frameIdentifier, sourceMapURL, sourceMapLoaded);
     }
 
-    _sourceMapLoadAndParseFailed(sourceMapURL)
+    _sourceMapLoadFailed(sourceMapURL)
     {
         this._downloadingSourceMaps.delete(sourceMapURL);
         this._failedSourceMapURLs.add(sourceMapURL);
     }
 
+    _sourceMapParseFailed(sourceMapURL, error)
+    {
+        this._downloadingSourceMaps.delete(sourceMapURL);
+        this._failedSourceMapURLs.add(sourceMapURL);
+
+        if (window.InspectorTest)
+            sourceMapURL = parseURL(sourceMapURL).lastPathComponent;
+
+        let message = WI.UIString("Source Map \u0022%s\u0022 has %s").format(sourceMapURL, error);
```

Comment:
const here and below? Or is that against the style guide? (It does seem like we use it in the last file so we should maybe sprinkle it around a bit more.)

Reply:
Web Inspector style is to only use `const` for things that do not change between sessions (i.e. a true global forever constant)